fail2ban/server/banmanager.py

```diff
@@ -26,6 +26,9 @@
 
 from threading import Lock
 
+import dns.exception
```

Comment:
The only issue with that approach is that with a munin-plugin (or other external program) parsing that data would be misleading. It is human useful, but blending the error message into the output isn't computer readable friendly imho